resources/knative-provisioner-natss/values.yaml

```diff
@@ -10,8 +10,8 @@ knative-provisioner-natss:
   controller:
     image:
       pullPolicy: IfNotPresent
-      location: gcr.io/knative-releases/github.com/knative/eventing/contrib/natss/cmd/channel_controller@sha256:943e9ba03f1f7536224f51bfb5181039c773bd4a2209574f44a385d9748ac97f
+      location: gcr.io/knative-releases/knative.dev/eventing-contrib/natss/cmd/channel_controller@sha256:3758dc790e862bac8098b469f3d37b4747fd6a5fb4e3a1f31bad1864e638c553
   dispatcher:
     image:
       pullPolicy: IfNotPresent
-      location: gcr.io/knative-releases/github.com/knative/eventing/contrib/natss/cmd/channel_dispatcher@sha256:4a5b9a5eb2271312cfec1d3808021a82def6b467273c09d7f8d0e762afefed82
+      location: gcr.io/knative-releases/knative.dev/eventing-contrib/natss/cmd/channel_dispatcher@sha256:cef2fbeff9e55b03b2b79e6b3e2db107645293cc2a86e3f25cf43b5df4b06f60
```

Comment:
We need to wait until https://github.com/kyma-project/kyma/issues/6310 is implemented. Otherwise we will reintroduce this bug: https://github.com/kyma-project/kyma/issues/6233

Reply:
This is no longer a blocker since the fix is available in eventing-contrib release [0.10.3](https://github.com/knative/eventing-contrib/compare/v0.10.2...v0.10.3).

Reply:
@marcobebway @nachtmaar Already pushed the last released images